web/src/main/java/org/springframework/security/web/csrf/CookieCsrfTokenRepository.java

```diff
@@ -220,4 +223,16 @@ public void setCookieMaxAge(int cookieMaxAge) {
 		this.cookieMaxAge = cookieMaxAge;
 	}
 
+	/**
+	 * Add the "SameSite" attribute to the generated cookie.
+	 * <p>This limits the scope of the cookie such that it will only be
+	 * attached to same site requests if {@code "Strict"} or cross-site
+	 * requests if {@code "Lax"}.
+	 * @since 6.1
+	 * @see <a href="https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis#section-4.1.2.7">RFC6265 bis</a>
+	 */
+	public void setSameSite(String sameSite) {
+		this.sameSite = sameSite;
```

Comment:
It would be nice if this same change were made in `CookieServerCsrfTokenRepository` (the reactive equivalent).

Reply:
I agree. I ported the changes to `CookieServerCsrfTokenRepository ` and added them to the PR.